### ISSUE TITLE
Add file size to data entries

### ISSUE THREAD

**Issue by paulschreiber** [techmatters/terraso-backend#167]
## Description
We need to display the size of each file. 
* Add a file size (integer number of bytes) to the shared data model 
* Expose this via the GraphQL API (fields in `shared_data.py`).
* Ensure the file size is populated at upload time

**Comment by paulschreiber** [comment]
@cacarrara this is done, right?

**Comment by cacarrara** [comment]
Yes. It was done here: https://github.com/techmatters/terraso-backend/pull/166/commits/ea440a250d6ea91dd4fe3d0ea734967d09118d7f
